src/cpp/include/openvino/genai/whisper_pipeline.hpp

```diff
@@ -12,37 +12,25 @@
 #include "openvino/genai/llm_pipeline.hpp"
 #include "openvino/genai/whisper_generation_config.hpp"
 
-namespace ov::genai {
+namespace ov {
+namespace genai {
 
 using OptionalWhisperGenerationConfig = std::optional<WhisperGenerationConfig>;
 
 using RawSpeechInput = std::vector<float>;
 
 /**
- * @brief base class for chunk streamers. In order to use inherit from from this class and implement put, and methods
- *
- * @param m_tokenizer tokenizer
+ * Base class for chunk streamers. In order to use inherit from from this class and implement put, and methods
  */
-class OPENVINO_GENAI_EXPORTS ChunkStreamerBase : public StreamerBase {
+class OPENVINO_GENAI_EXPORTS OPENVINO_DEPRECATED(
```

Comment:
we need to avoid header only base classes. 
Please, override dtor for this class and put its definitionto .cpp file

Reply:
Done